plugins/pipelines/startAll.js

```diff
@@ -0,0 +1,79 @@
+'use strict';
+
+const boom = require('boom');
+const joi = require('joi');
+const schema = require('screwdriver-data-schema');
+const idSchema = joi.reach(schema.models.pipeline.base, 'id');
+
+module.exports = () => ({
+    method: 'POST',
+    path: '/pipelines/{id}/startall',
```

Comment:
camel case maybe? `startAll`?

Reply:
don't think it's good to have camel case in url...

Reply:
good point whoops